addons/marketing_card/security/marketing_card_groups.xml

```diff
@@ -1,20 +1,20 @@
 <?xml version="1.0"?>
 <odoo noupdate="1">
-    <record id="module_category_marketing_card" model="ir.module.category">
+    <record id="base.module_category_marketing_marketing_card" model="ir.module.category">
```

Comment:
Hello @thsh-odoo,
The commit/PR message mentioned that it does not work without the `base` prefix but why?

Is it because the `base` prefix forces the creation of a new record in db? Shouldn't we rather force the update of the record in the migration scripts?

Reply:
In the module category record, we use the "base" prefix to ensure that the specified sequence in the record is respected. Without the "base" prefix, the sequence would not be followed. This convention is applied consistently throughout the Odoo codebase.

I’ve observed this pattern in every module, and I implemented it because I was facing sequencing issues that were resolved by using the "base" prefix.

Reply:
I'm not an expert about that but if it works and if we also use the `base` prefix in the other modules to set the module description, sequence number, etc. it should be okay :smiley:

Reply:
Hello @aj-fuentes 
Actually, it totally depends on this
https://github.com/odoo/odoo/pull/176922/files#diff-190743079f53fc0e78c5c5aa6aca4d7709b3143eb7a61ed331aba261f0a2a41bR4